doc/set_fees_process.md

```diff
@@ -0,0 +1,40 @@
+# Setting fees process
+
+It will be explained on an example of Indy CLI.
+
+### Prerequisites
+
+* libindy
+* Indy CLI
+* libsovtoken
+* Wallet with Trustee DID
+* Created pool in Indy CLI
+
+[Small guide](https://github.com/hyperledger/indy-sdk/tree/master/doc/design/001-cli#commands) to Indy CLI commands: 
+
+### Creating a SET_FEES transaction.
+
+* Open `indy-cli`
+* Open wallet: `wallet open <wallet_name> key=<wallet_encryption_key>`
```

Comment:
Verifying: they don't need to connect to the pool here because there is no communication with the ledger?

Reply:
Yes, because of that.